src/bindings/python/tests/test_runtime/test_function.py

```diff
@@ -448,3 +450,68 @@ def check_shape(new_shape):
         "expected values as openvino.runtime.PartialShape, str, list or tuple."
         in str(e.value)
     )
+
+
+def test_serialize_rt_info():
+    version = "TestVersion"
+    config = "TestConfig"
+    framework_batch = "1"
+
+    def check_rt_info(model):
+        assert model.get_rt_info(["MO_version"]) == version
+        assert model.get_rt_info(["Runtime_version"]) == version
+        assert model.get_rt_info(["optimization", "config"]) == config
+        assert model.get_rt_info(["framework", "batch"]) == framework_batch
+
+        assert model.has_rt_info(["test"]) is False
+        assert model.has_rt_info(["optimization"]) is True
+        assert model.has_rt_info(["optimization", "test"]) is False
+        with pytest.raises(RuntimeError):
+            assert model.get_rt_info(["test"])
+
+        with pytest.raises(RuntimeError):
+            assert model.get_rt_info(["optimization", "test"])
+
+    core = Core()
+    xml_path = "./serialized_model.xml"
+    bin_path = "./serialized_model.bin"
+    input_shape = PartialShape([1])
+    param = ops.parameter(input_shape, dtype=np.float32, name="data")
+    relu1 = ops.relu(param, name="relu1")
+    relu1.get_output_tensor(0).set_names({"relu_t1"})
+    assert "relu_t1" in relu1.get_output_tensor(0).names
+    relu2 = ops.relu(relu1, name="relu2")
+    model = Model(relu2, [param], "TestFunction")
+
+    assert model is not None
+
+    assert model.has_rt_info(["MO_version"]) is False
+    model.set_rt_info(version, ["MO_version"])
+    assert model.has_rt_info(["MO_version"]) is True
+
+    assert model.has_rt_info(["Runtime_version"]) is False
+    model.set_rt_info(version, ["Runtime_version"])
+    assert model.has_rt_info(["Runtime_version"]) is True
+
+    assert model.has_rt_info(["optimization"]) is False
+    assert model.has_rt_info(["optimization", "config"]) is False
+    model.set_rt_info(config, ["optimization", "config"])
+    assert model.has_rt_info(["optimization"]) is True
+    assert model.has_rt_info(["optimization", "config"]) is True
+
+    assert model.has_rt_info(["framework"]) is False
+    assert model.has_rt_info(["framework", "batch"]) is False
+    model.set_rt_info(framework_batch, ["framework", "batch"])
+    assert model.has_rt_info(["framework"]) is True
+    assert model.has_rt_info(["framework", "batch"]) is True
+
+    check_rt_info(model)
+
+    serialize(model, xml_path, bin_path)
+
+    res_model = core.read_model(model=xml_path, weights=bin_path)
+
```

Comment:
I think before serialization and after serialization and reading we should perform the same check. Can you do this?

Reply:
@rkazants I guess that `check_rt_info` do this thing.

Reply:
Why do you have several `assert` checks above? I mean to use the same scope of checks before and after.